core/domain/feedback_domain.py

```diff
@@ -37,9 +56,20 @@ def __init__(self, full_thread_id, exploration_id, state_name,
         self.last_updated = last_updated
 
     def get_thread_id(self):
+        """
+        
```

Comment:
Yes, remove the line. But where's the docstring? There should be a (usually one-line) description of what the method does at the top of the docstring.